cluster/gce/trusty/node.yaml

```diff
@@ -133,12 +131,15 @@ Content-Disposition: attachment; filename="kube-install-additional-packages.conf
 
 description "Install additional packages used by kubernetes"
 
-start on started docker
+start on stopped kube-install-packages
 
 script
```

Comment:
This file contains a bunch of upstart jobs, each of which is in shell-like syntax. But it does not support all bash functionality. I can verify if these "set -o" commands work in upstart, and add them if they do.